source/develop/release-management/features/infra/Engine_vacuum.md

```diff
@@ -0,0 +1,123 @@
+# Engine Vacuum
+
+# Description
+This page describes the **_engine-vacuum_** cli tool and the **_vacuum setup-plugin_** that uses it.
+
+# Motivation
+
+> "PostgreSQL databases require periodic maintenance known as vacuuming.
+For many installations, it is sufficient to let vacuuming be performed by the autovacuum daemon"
+> - routine-vacuummin, [postgres documentation][1]
+
+Apart from updating the tables and removing garbage from the tables,
+full vacuum is the only vacuum operation that can reclaim disk space back to the operating system.
+If the engine db is falling behind in collecting garbage,(autovacuum doesn't perfom optimally) we will
+see disk growth in time as a result.
+
+The engine-vacuum tool is a wrapper around postgres commands and is aimed to ease Vacuum
+maintenance actions on the DB, and can be used easily and securely, as they reuse
+the setup credentials to authenticate against the engine db. The tool is being
+used by an engine-setup plugin and act as a maintenance task, taking advantage of
+the downtime period when updating the engine. The tool can be used just as well
+by cron, script or just manual invocation and it covers most of the functionality
+of the vacuum commands.
+
+# Scope
+This page covers the tool itself, and the engine-setup plugin that was created to use it.
+
+Not covered - Vacuum design and functionality. See the references section to read about vacuum.
+
+# Synopsis
+## engine-vacuum
+  engine-vacuum [OPTION]...
+
+Located under `/bin/engine-vacuum`, invoking this can perform vacuum, vacuum full
+and analyze against the installed engine db, optionally specifying the tables,
+without needing to specifying the engine db, user and password.
+```text
+engine-vacuum --help
+
+  -a          - run analyze, update optimizer statistics
```

Comment:
added links inline (in the examples section) and added the relevant wiki page at the reference section